js/glotdict-consistency.js

```diff
@@ -1,160 +1,161 @@
-if ( typeof $gp !== 'undefined' ){
-  var gd_quicklinks_copy_state = localStorage.getItem( 'gd_quicklinks_copy_state' ) === 'true';
-  var gd_quicklinks_window = { 'closed': true };
-  gd_quicklinks();
-  gd_consistency();
+let gd_quicklinks_copy_state = 'true' === localStorage.getItem( 'gd_quicklinks_copy_state' );
+let gd_quicklinks_window = { 'closed': true };
+
+if ( typeof 'undefined' !== typeof $gp ) {
```

Comment:
typeof 'undefined' is 'string'
typeof $gp is either 'object' or 'undefined'

These will never be equal...

Reply:
Yep that's a mistake I did and just understood that!
I make a commit